### DISCUSSION TITLE
[FEATURE-REQUEST] Feedback de usuario: nuevo formato de campos de hora es difícil de usar

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#214] [3. Ideas & Feature Requests] [answered]
Buenas,

Un cliente que tiene una implementación que se está utilizando hace unos meses reporta que con el último cambio que se hizo a los campos de hora, provoca que sea más difícil indicar una hora específica en el reloj. Usuario indica que la precisión de 5 minutos no es suficiente. Sólo se puede indicar mediante la interfaz horas de 5 en 5 minutos a menos que se escriba y esto les parece peor experiencia de usuario que la manera anterior que se tenía.

Se propone evaluar el prellenado automático de la hora actual para ver si esto facilita el ingreso de este dato para los usuarios finales. Cuando lo prueben, tendremos feedback pero no soluciona totalmente el problema. ¿Podemos evaluar decidir si utilizar la nueva interfaz o la antigua en la misma question en el survey builder del administrador? ¿Hay otra alternativa que le pueda dar al cliente? Como antecedente, el cliente indica que: "los usuarios tienen más de 50 años y no siempre le achuntan a las horas en el reloj".

Saludos.

**Comment by guillevalin** [comment, marked as answer]
@SeIturra Recibimos el feedback. Podríamos hacer mejoras en este selector.

Si esto es algo prioritario para alguna implementación, te solicitamos por favor canalizar esta solicitud a través de tu representante de Cotalker.

**Reply by seiturra** [reply]
Gracias @guillevalin!
